showpass/Showpass-E2E-tests
# Ta 44 enable notifications

**Description:**
Create a new method which will be executed before the 'Notifications' are tested. This method will be checking if any notifications are disabled before the main test gets executed and will be turning on any disabled notification found.
- Register a new user to test notification selectors only
- Add a method to turn back on the disabled notification if any
- Ensure the entire ‘Profile’ test suite passes
- Try to run various edge cases to ensure consistency

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-44

**How to test:**
Modify 'Notifications' selectors as per your choice and save changes. Try to turn off them randomly. Save changes.
After some selectors are disabled run the following specs under the 'Profile' test suite and ensure all of them pass.
- verifying that notifications can be turned on and off-TA-22
- verifying that notifications toggles disappear once 'All Notifications' selector is off-TA-24
- verifying that updated notifications can be saved-TA-25

**Test execution results:**

<img width="748" alt="TA-44-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/06a9b972-27eb-43c7-831f-e2d91b611bfe">
